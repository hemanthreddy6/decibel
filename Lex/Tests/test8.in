my_audio_1: audio = audio(SIN,1.2,120);
my_audio_2: audio = audio(COS,1.2,120);

my_audio_1 = my_audio_1 + my_audio_2;
sliced_audio = my_audio_1[:23];

main {
    play sliced_audio^12;  // plays the file 12 times 
}
