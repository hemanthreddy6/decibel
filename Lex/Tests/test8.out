AUDIO
IDENTIFIER
=
AUDIO
(
SIN
,
FLOAT_LITERAL
,
INT_LITERAL
)
;
AUDIO
IDENTIFIER
=
AUDIO
(
COS
,
FLOAT_LITERAL
,
INT_LITERAL
)
;
IDENTIFIER
=
IDENTIFIER
+
IDENTIFIER
;
IDENTIFIER
=
IDENTIFIER
[
:
INT_LITERAL
]
;
PLAY
IDENTIFIER
^
INT_LITERAL
;
